src/devinterp/optim/sgld.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         temperature: Union[Callable, float] = 1.0,
         bounding_box_size=None,
         save_noise=False,
+        optimize_over=None,
```

Comment:
i think this parameter is no longer used and instead this is in `params` which is updated by the sampler? if so we can delete it (and from the default params below)